layouts/episodes/single.html

```diff
@@ -14,7 +14,7 @@ <h2 id="show-notes">Show Notes</h2>
                     'replace' below is to ensure correct local link across local, dev, and prod URLs.
                     Front matter does not allow variables, so we need to do string substitution.
                 */}}
-                <p>{{ (replace .Params.showNotes "!!baseurl!!" .Site.BaseURL) | markdownify }}</p>
+                <p>{{ (replace .Params.shownotes "!!baseurl!!" .Site.BaseURL) | markdownify }}</p>
```

Comment:
Same here - don't think we need this wrapper `<p>` tag.

Reply:
✅